### ISSUE TITLE
Can't load containers addon with about:debugging. FF57.

### ISSUE THREAD

**Issue by falolaf** [mozilla/multi-account-containers#965]
Hello,

I have for the past couple of months used my own version of the containers addon by adding it via about:debugging. But with the release of FF57 this now fails. When trying to load I get this message:
There was an error during installation: Add-on @testpilot-containers is not compatible with application version. add-on minVersion: 53.0. add-on maxVersion: *.

I have two of my own commits ontop of latest master and that have worked without problems with FF55 and FF56.
I have tried to change the min- and maxVersion but to no avail.
Also tested with a fresh profile.

The addon from amo loads just fine it seems though.

Any thoughts about this?

**Comment by KwanEsq** [comment]
@falolaf do you have the same problem if you set extensions.legacy.enabled to true in about:config?

**Comment by jonathanKingston** [comment]
@falolaf have you instead just loaded the web extension part as mentioned in the readme? We likely will delete the other code now 57 is stable.

**Comment by falolaf** [comment]
@KwanEsq Same problem even though that parameter is set.

**Comment by falolaf** [comment]
@jonathanKingston If you mean to load with "web-ext run", I have tested that but it seems that all tabs in all containers are lost when loading this way. I have reverted to 56.0.2 again and everything is working as expected.

Any idea why it is possible to install the addon from amo but not my own local one? It is the same code base I assume.

**Comment by falolaf** [comment]
@jonathanKingston I might have misunderstood you. I have now loaded the webextension part of the addon in about:debugging buy loading any file in the webextension folder and that works. I had to add "id": "@testpilot-containers" under "application" "gecko" in webextension/manifest.json though to replace the legacy addon and be able to see the same containers. 

Now my own changes seem to work so I am happy again, using FF57.
